package.json

```diff
@@ -28,5 +28,8 @@
   "author": {
     "name": "Amazon Web Services",
     "url": "https://aws.amazon.com"
+  },
+  "dependencies": {
+    "@aws-cdk/aws-sqs": "^0.32.0"
```

Comment:
ha?

Reply:
This is a typo from my side, I just checked the history